utils/buttons.py

```diff
@@ -66,7 +99,7 @@ class ViewButtonIteration(BaseView):
     """A BaseView class that creates arrays of buttons, depending on the data type given on 'args',
         it will accept `mapper` as a dataset"""
     def __init__(self, *args: Any, mapper: Optional[Dict[str, Any]] = None,
-                 button: Optional[Type[BaseButton]] = BaseButton, style: Optional[discord.ButtonStyle] = None):
```

Comment:
why is style no longer Optional

Reply:
because `ui.Button` style and `BaseButton` do not accept None

Reply:
There is a case where style can be None so false

Reply:
wdym a case where it can be none? im just looking at type hints and there is no Optional for style

Reply:
source: trust me bro 
[dpy source](https://github.com/Rapptz/discord.py/blob/master/discord/ui/button.py#L108)

Reply:
now dont cry please

Reply:
this is implementation detal that ignores type of style. function signature does not allow None. i agree that it will work currently but it is not a clean solution to put None there

Reply:
look, I dont like redundant interfaces
Doing this
```python
urls = [{'url': 'http1'}, {'url': 'http2'}]
ViewButtonIteration(urls, style=discord.ButtonStyle.link)
```
Instead of this
```python
urls = [{'url': 'http1'}, {'url': 'http2'}]
ViewButtonIteration(urls)
```
You keep crying about function signature when I wanted a good function interface.